### DISCUSSION TITLE
Cola, an serverless way to write code in parallel.

### DISCUSSION THREAD

**Discussion by koan09** [neovim/neovim#25240] [Ideas]
Hi, I was wondering if it would be hard to implement something like cola. [https://github.com/nomad/cola] So I can work simultaneously with other ppl on text files.

**Comment by justinmk** [comment]
[floobits](https://github.com/Floobits) was an early product that did this. Not sure about its status.

You could also use vscode (which supports collaborative editing) with the [vscode-neovim](https://github.com/vscode-neovim/vscode-neovim) plugin.
